packages/url_launcher/url_launcher_macos/macos/Classes/UrlLauncherPlugin.swift

```diff
@@ -5,7 +5,21 @@
 import FlutterMacOS
 import Foundation
 
+public protocol URLLauncher {
```

Comment:
This is a potentially confusing name given that it's the name of the plugin. How about `SystemURLHandler`?